### ISSUE TITLE
Logs spam in ethdocker and sedge

### ISSUE THREAD

**Issue by MarekM25** [NethermindEth/nethermind#6017]
Starting from 1.20 we have lots of spam in logs in ethdocker and we have it in sedge as well. Here is how it looks for eth_docker

![image](https://github.com/NethermindEth/nethermind/assets/9356351/94a494c1-d2f3-4f8d-9a25-a83cbe4ce650)

And how it should look (even without colors):
![image](https://github.com/NethermindEth/nethermind/assets/9356351/a97d06c5-e746-4f01-9156-d8dd0082b798)

We should check NLog configurations for sedge and ethdocker

FYI: @yorickdowne

**Comment by yorickdowne** [comment]
I am using the NLog as it’s in the image, without any adjustments made. Thanks for taking a look!

**Comment by smartprogrammer93** [comment]
could it be something to do with the way we create our docker images?

**Comment by yorickdowne** [comment]
Possibly - the NLog.Config is in the image. Does it differ in the image to how it would otherwise be deployed?

**Comment by yorickdowne** [comment]
In case that helps, here is the NLog.config as it is shipped in the image

[NLog.config.txt](https://github.com/NethermindEth/nethermind/files/12597007/NLog.config.txt)

**Comment by yorickdowne** [comment]
Looking through that config, here's what I don't see. If this existed, it would likely stop the RPC spam.

```
  <rules>
    <logger name="JsonRpc.*" minlevel="Warn" writeTo="file-async" final="true" />
    <logger name="JsonRpc.*" minlevel="Warn" writeTo="auto-colored-console-async" final="true" />
    <logger name="JsonRpc.*" final="true" />
```

It may be as simple as adding rules for `"JsonRpc.*"`

**Comment by emlautarom1** [comment]
We were able to find the root cause and a fix is already available on #6097. With this change you should not see any more spam related to HTTP calls.
